tests/test_runner/test_discover_runner.py

```diff
@@ -765,6 +766,32 @@ def test_suite_result_with_failure(self):
                 failures = runner.suite_result(suite, result)
                 self.assertEqual(failures, expected_failures)
 
+    @unittest.skipUnless(PY312, "Durations added in Python 3.12")
+    def test_durations(self):
+        with captured_stderr() as stderr, captured_stdout():
+            runner = DiscoverRunner(durations=10)
+            suite = runner.build_suite(["test_runner_apps.simple.tests.SimpleCase1"])
+            runner.run_suite(suite)
+        self.assertIn("Slowest test durations", stderr.getvalue())
+
+    @unittest.skipUnless(PY312, "Durations added in Python 3.12")
+    def test_durations_parallel(self):
+        with captured_stderr() as stderr, captured_stdout():
+            runner = DiscoverRunner(durations=10, parallel=2)
+            runner.parallel_test_suite.serialized_aliases = [""]
+            suite = runner.build_suite(["test_runner_apps.simple.tests"])
+            runner.run_suite(suite)
```

Comment:
This crashes on Python 3.12.0rc1 with :imp: :no_entry_sign:  :baby: :
```
AssertionError: daemonic processes are not allowed to have children
```

Reply:
I've pushed a commit, but need to investigate if this still actually gives the coverage we're looking for...

Reply:
I'm sorry, I'm not sure how to fix this test. 

Do you think it's enough to test `RemoteTestResult` alone. Something like https://github.com/django/django/pull/12580/files#diff-3a08c8ec995b25f8c7acbcabdd074337d5ebc246e429037c0e3a363d077ccbb3

Reply:
> Do you think it's enough to test `RemoteTestResult` alone.

Should be fine.

Reply:
@smithdc1 Do you have time to keep working on this?